docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -41,6 +41,28 @@ The `dagster-dbt` library installs both `dbt-core` and `dagster` as dependencies
 
 ---
 
+## Wrap your dbt project with Dagster
+
+<TabGroup>
+<TabItem name="Option 1: Create a brand new Dagster project">
+
+### Option 1: Create a brand new Dagster project
+
+You can create a Dagster project that wraps your dbt project by using the `dagster-dbt` command line interface.
+
+To use the command, you'll need to provide 2 options - `--project-name`, the name of your Dagster project to be created, and `--dbt-project-dir`, the path to your dbt project. In our example, our Dagster project is named `my-dagster-project`and the relative path to our dbt project is `./my-dbt-project`, meaning that we are in the directory where `my-dbt-project` is located.
```

Comment:
We can change this so that the user only has to provide one parameter `--dbt-project-dir`.

We can do this by reading the user's `dbt_project.yml` and using the current name as the project name. Maybe event append `_dagster` to that dbt project name,

Reply:
Currently, if we don't provide the `--project-name` parameter, the user is prompted for the project name.  Should we update this behaviour?

If so, could we do it in another PR? My goal here was to make the command as explicit as possible to make it easier for the user to understand.

Reply:
Yeah we should do it in another PR.